meeting-notes/2018/2018-11-21--gui-and-in-web-browsers-weekly.md

```diff
@@ -0,0 +1,127 @@
+# GUI & In Web Browsers Weekly Sync 2018-11-21
+
+- **Lead:** @olizilla
+- **Notetaker:** @lidel
+- **Attendees:**
+    - @terichadbourne
+    - @olizilla
+    - @lidel
+    - @alanshaw
+    - @hacdias
+    - fsdiogo
```

Comment:
Add missing `@` please!

```suggestion
    - @fsdiogo
```

Reply:
@fsdiogo it's simplest if you just jump in here and be the change you want to see

Reply:
For sure, but I can't edit the PR, probably due to some permission settings? 🤷‍♂️